ext/native/thin3d/VulkanRenderManager.cpp

```diff
@@ -1141,13 +1138,22 @@ void VulkanRenderManager::EndSubmitFrame(int frame) {
 		present.waitSemaphoreCount = 1;
 
 		VkResult res = vkQueuePresentKHR(vulkan_->GetGraphicsQueue(), &present);
-		// TODO: Deal with VK_SUBOPTIMAL_KHR ?
-		if (res == VK_ERROR_OUT_OF_DATE_KHR || res == VK_SUBOPTIMAL_KHR) {
-			// ignore, it'll be fine. this happens sometimes during resizes, and we do make sure to recreate the swap chain.
+		// TODO: Deal with VK_SUBOPTIMAL_KHR here too?
```

Comment:
We at least check for it and increment, is there more left for this TODO?

-[Unknown]

Reply:
Hm, no not really, editing mistake.